recipes/qt/5.x.x/conanfile.py

```diff
@@ -661,6 +661,16 @@ def package(self):
         # symbols that are also in "Qt5Core.lib". It looks like there is no "Qt5Bootstrap.dll".
         for fl in glob.glob(os.path.join(self.package_folder, "lib", "*Qt5Bootstrap*")):
             os.remove(fl)
+        tools.save(os.path.join(self.package_folder,"lib", "cmake", "Qt5Core", "extras.cmake"),
+                    textwrap.dedent("""\
+                        set(Qt5Core_QMAKE_EXECUTABLE ${CMAKE_CURRENT_LIST_DIR}/../../../bin/qmake)
+                        set(Qt5Core_MOC_EXECUTABLE ${CMAKE_CURRENT_LIST_DIR}/../../../bin/moc)
+                        set(Qt5Core_RCC_EXECUTABLE ${CMAKE_CURRENT_LIST_DIR}/../../../bin/rcc)
+                        set(Qt5Core_UIC_EXECUTABLE ${CMAKE_CURRENT_LIST_DIR}/../../../bin/uic)"""))
+        for m in os.listdir(os.path.join(self.package_folder, "lib", "cmake")):
+            module = os.path.join(self.package_folder, "lib", "cmake", m, "%sMacros.cmake" % m)
+            if not os.path.isfile(module):
+                tools.rmdir(os.path.join(self.package_folder, "lib", "cmake", m))
```

Comment:
What happen here is not obvious, don't you want to remove these folders before creating `extras.cmake`?

Reply:
This is not correlated to extras.cmake. This loop just removes cmake folders which don't contain anything useful. I don't get what is accomplished by removing these useless folders  before creating `extras.cmake` (which I'm renaming now). Could you please clarify ?

Reply:
Just looking at the code, how to you know that the folder containing `extras.cmake` is not removed by this block? It's not obvious.